### PR TITLE
Support for scaling up etcd

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://trello.com/c/EESwIsuW/171-5-support-for-scaling-up-etcd

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
[Document: Support for scaling up etcd](https://trello.com/c/hp6BVVct/628-document-support-for-scaling-up-etcd)

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 1]
```diff
@@ -250,3 +250,47 @@ infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'defa
...
++
+For example, to add a new etcd, add *new_etcd*:
++
+----
+[OSEv3:children]
+masters
+nodes
+new_etcd
+----
+
```
Comment (gpei) [inline install_config/adding_hosts_to_existing_cluster.adoc]: Also need to add new etcd host to children group [new_etcd], like:

[new_etcd]
new_etcd1.example.com

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 2]
```diff
@@ -250,3 +250,48 @@ infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'defa
...
+*[OSEv3:children]* group and add hosts under the *new_<host_type>* group:
++
+For example, to add a new etcd, add *new_etcd*:
++
+----
+[OSEv3:children]
+masters
+nodes
+[new_etcd]
+new_etcd1.example.com
```
Comment (abutcher) [inline install_config/adding_hosts_to_existing_cluster.adoc]: `etcd` and `new_etcd` should be added to `[OSEv3:children]` in this example and I'd maybe fill out the etcd group just to show a little more.

```
[OSEv3:children]
masters
nodes
etcd
new_etcd

[etcd]
etcd1.example.com
etcd2.example.com

[new_etcd]
etcd3.example.com
```

**Comment by gaurav-nelson** [comment]
thanks @abutcher I have added suggested example

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL

**Comment by gaurav-nelson** [comment]
[rev_history]
|xref:../install_config/adding_hosts_to_existing_cluster.adoc#install-config-adding-hosts-to-cluster[Adding Hosts to an Existing Cluster]
|Added the xref:../install_config/adding_hosts_to_existing_cluster.adoc#adding-etcd-hosts-to-existing-cluster[Adding etcd Hosts to existing Cluster] section.
%

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 3]
```diff
@@ -250,3 +250,55 @@ infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'defa
...
+new hosts only. Before running the etcd  *_scaleup.yml_* playbook, complete all
+prerequisite
+xref:../install_config/install/host_preparation.adoc#install-config-install-host-preparation[host
+preparation] steps.
+
+To add an etcd host to an existing cluster:
+
+. Ensure you have the latest playbooks by updating the *atomic-openshift-utils* package:
++
+
```
Comment (ahardin-rh) [inline install_config/adding_hosts_to_existing_cluster.adoc]: Extra line here between the + and the command

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 4]
```diff
@@ -250,3 +250,55 @@ infra-node2.example.com openshift_node_labels="{'region': 'infra', 'zone': 'defa
...
+etcd1.example.com
+etcd2.example.com
+
+[new_etcd]
+etcd3.example.com
+----
+
+. Run the etcd *_scaleup.yml_* playbook. If your inventory file is located somewhere other than the default of *_/etc/ansible/hosts_*, specify the location with the `-i` option.
++
+
```
Comment (ahardin-rh) [inline install_config/adding_hosts_to_existing_cluster.adoc]: Same here. I think the extra line will prevent keeping the code block inline.

**Comment by ahardin-rh** [comment]
@gaurav-nelson just one minor bit of feedback from me. Otherwise, looks good!

**Comment by gaurav-nelson** [comment]
Thanks @ahardin-rh, fixed extra line breaks in the latest commit.

**Pull request merged**